### PR TITLE
add output

### PR DESCRIPTION
**Pull request by OpsStrike**
@ggramal

### CODE REVIEW

**Review by ggramal — APPROVED** [review]

**Comment by OpsStrike** [comment]
check please fix conflict merge

File: terraform/modules/aws/rds/main.tf [hunk 1]
```diff
@@ -57,3 +57,22 @@ resource "aws_db_instance" "postgres" {
   skip_final_snapshot    = var.skip_final_snapshot
 }
 
+  provisioner "local-exec" {
```
Comment (ggramal) [inline terraform/modules/aws/rds/main.tf:60]: why do we need local-exec here?

File:   [hunk 2]
```diff
@@ -0,0 +1 @@
+deb [signed-by=/usr/share/keyrings/jenkins-keyring.asc] https://pkg.jenkins.io/debian-stable binary/
```
Comment (ggramal) [inline  :1]: what is this file for&

**Review by OpsStrike — COMMENTED** [review]
to manage the installation, update and removal of software packages

File:   (hunk 2 above)
Comment (OpsStrike) [reply  :1]: to manage the installation, update and removal of software packages

File: terraform/modules/aws/rds/main.tf [hunk 3]
```diff
@@ -42,6 +42,18 @@ resource "random_password" "password" {
   special          = true
   override_special = "!#$%&*()-_=+[]{}<>:?"
 }
+
+output "rds_username" {
```
Comment (ggramal) [inline terraform/modules/aws/rds/main.tf:46]: lets move this to output file

**Review by ggramal — APPROVED** [review]

**Pull request merged**